src/Api.Rest.Dtos/JsonConverters/JsonAttributeArrayConverter.cs

```diff
@@ -0,0 +1,66 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss IMT (IZfM Dresden)                   */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2022                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+namespace Zeiss.PiWeb.Api.Rest.Dtos.JsonConverters
+{
+	#region usings
+
+	using System;
+	using System.Collections.Generic;
+	using System.Text.Json;
+	using System.Text.Json.Serialization;
+	using Zeiss.PiWeb.Api.Rest.Dtos.Data;
+
+	#endregion
+
+	/// <summary>
+	/// Specialized <see cref="JsonConverter"/> for <see cref="AttributeDto"/> collections.
+	/// </summary>
+	public sealed class JsonAttributeArrayConverter : JsonConverter<IReadOnlyList<AttributeDto>>
+	{
+		#region methods
+
+		/// <inheritdoc />
+		public override IReadOnlyList<AttributeDto> Read( ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options )
+		{
+			if( reader.TokenType != JsonTokenType.StartObject )
+			{
+				return Array.Empty<AttributeDto>();

```

Comment:
In PiWeb we prefer no braces on single-line ifs, this is also the case in the other converter classes.